generated/efmls-configs.nix

```diff
@@ -49,7 +49,9 @@
   blade = {
     formatter = {
       lang = "blade";
-      possible = [ "blade_formatter" ];
+      possible = [
+        "blade_formatter"
+      ];
```

Comment:
Seems like `nix fmt` failed? The generate-files script is _supposed_ to format the generated files automatically...